src/Microsoft.DotNet.Darc/src/Darc/Helpers/UxHelpers.cs

```diff
@@ -220,6 +220,25 @@ public static string GetTextSubscriptionDescription(Subscription subscription, I
             return subInfo.ToString();
         }
 
+        public static string DependencyToString(DependencyDetail dependency)
```

Comment:
Would it make sense to override the ToString for DependencyDetail?

Reply:
I was trying to avoid touching something that might be used by product code, but I don't see any uses of it in arcade-services. The question is whether code in another repo is depending on a certain behavior for ToString, since it's a public class in DarcLib.

Reply:
I think it's fine either way